Core/Base/Client.py

```diff
@@ -1,57 +1,62 @@
-""" Base class for DIRAC Client """
+""" Module that expose the base class for DIRAC Clients.
+
+    This class exposes possible RPC calls, given a url of a service.
+"""
 
 __RCSID__ = "$Id$"
 
-from DIRAC.Core.DISET.RPCClient                     import RPCClient
+from DIRAC.Core.DISET.RPCClient import RPCClient
+
 
-class Client( object ):
+class Client(object):
   """ Simple class to redirect unknown actions directly to the server. Arguments
       to the constructor are passed to the RPCClient constructor as they are.
       Some of them can however be overwritten at each call (url and timeout).
       This class is not thread safe !
 
       - The self.serverURL member should be set by the inheriting class
   """
-  def __init__( self, **kwargs ):
+
+  def __init__(self, **kwargs):
     """ C'tor.
 
         :param kwargs: just stored as an attribute and passed when creating
                       the RPCClient
     """
     self.serverURL = None
-    self.call = None # I suppose it is initialized here to make pylint happy
+    self.call = None  # I suppose it is initialized here to make pylint happy
     self.__kwargs = kwargs
 
-  def setServer( self, url ):
+  def __getattr__(self, name):
```

Comment:
It did not change, did it ? Any reason to move it there ? :D

Reply:
Just for "order"... `__getattr__` after `__init__`